nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -384,69 +427,112 @@ These endpoints `/ipam/roles/`, `/dcim/rack-roles/` and `/dcim/device-roles/` ar
+|                         | `cluster_type`            | Enhanced to support primary key UUIDs in addition to names | `/virtualization/clusters/?cluster_type=<uuid/name>`           |
+|                         | `tenant`                  | Enhanced to support primary key UUIDs in addition to names | `/virtualization/clusters/?tenant=<uuid/name>`                 |
+|                         | `tenant_group`            | Enhanced to support primary key UUIDs in addition to names | `/virtualization/clusters/?tenant_group=<uuid/name>`           |
+| ConfigContext           | `cluster_group`           | Enhanced to support primary key UUIDs in addition to names | `/extras/config-contexts/?cluster_group=<uuid/name>`           |
+|                         | `device_redundancy_group` | Enhanced to support primary key UUIDs in addition to names | `/extras/config-contexts/?device_redundancy_group=<uuid/name>` |
+|                         | `platform`                | Enhanced to support primary key UUIDs in addition to names | `/extras/config-contexts/?platform=<uuid/name>`                |
+|                         | `tenant`                  | Enhanced to support primary key UUIDs in addition to names | `/extras/config-contexts/?tenant=<uuid/name>`                  |
+|                         | `tenant_group`            | Enhanced to support primary key UUIDs in addition to names | `/extras/config-contexts/?tenant_group=<uuid/name>`            |
+|                         | `dynamic_groups`          | Enhanced to support primary key UUIDs in addition to names | `/extras/config-contexts/?dynamic_groups=<uuid/name>`          |
+| ConsolePort             | `device`                  | Enhanced to support primary key UUIDs in addition to names | `/dcim/console-ports/?device=<uuid/name>`                      |
+| ConsoleServerPort       | `device`                  | Enhanced to support primary key UUIDs in addition to names | `/dcim/console-server-ports/?device=<uuid/name>`               |
+| Device                  | `cluster_id`              | Enhanced to support primary key UUIDs in addition to slugs | `/dcim/devices/?cluster=<uuid/slug>`                           |
+|                         | `device_type_id`          | Enhanced to support primary key UUIDs in addition to slugs | `/dcim/devices/?device_type=<uuid/slug>`                       |
+|                         | `device_redundancy_group` | Enhanced to support primary key UUIDs in addition to names | `/dcim/devices/?device_redundancy_group=<uuid/name>`           |
+|                         | `manufacturer`            | Enhanced to support primary key UUIDs in addition to names | `/dcim/devices/?manufacturer=<uuid/name>`                      |
+|                         | `model`                   | Enhanced to support primary key UUIDs in addition to slugs | `/dcim/devices/?model=<uuid/slug>`                             |
```

Comment:
Should this be names instead of slugs?

Reply:
I dont think Device has model filter anymore, so i removed it